src/user_model.py

```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, Integer, String
+
+class User(Base):
```

Comment:
Hey Adi,

We see you'v added a new DB table to the project.

Is this table going to contain some PIIs ?

Reply:
No